### ISSUE TITLE
`pyenv-virtualenv-init` not available on PATH

### ISSUE THREAD

**Issue by agoose77** [ohmyzsh/ohmyzsh#8467]
In my version of pyenv (1.2.14), there is no executable named `pyenv-virtualenv-init` on the search `$PATH`, and hence there is no entry in `$commands`. It follows that the virtualenv machinery is not initialised here: https://github.com/ohmyzsh/ohmyzsh/blob/664664f6cd8a1c02a38063c8f2104f0515a19399/plugins/pyenv/pyenv.plugin.zsh#L28

I don't know about the best way to solve this. Several approaches include
- Running `pyenv virtualenv --version` and catching an error code
- Running `pyenv commands` and grepping for virtualenv.

**Comment by mcornella** [comment]
Have you checked that after running the previous line (`eval "$(pyenv init - zsh)"`)?

**Comment by mcornella** [comment]
Also, I assume you have installed the [pyenv-virtualenv plugin](https://github.com/pyenv/pyenv-virtualenv)?

**Comment by agoose77** [comment]
Hi @mcornella,
I've removed the explicit initialisation and replaced it with the oh-my-zsh plugin, which is when I noticed the problem. I have the plugin available (it's visible from `pyenv commands`), but it seems that (in my version at least) the actual binaries of the plugin are never placed on the path. This seems like a design decision to me, as it means that plugins are all unified under the pyenv namespace.
